### PR TITLE
New Package: Spandoc

### PR DESCRIPTION
**Pull request by geniusupgrader**

### CODE REVIEW

**Comment by geniusupgrader** [comment]
I have a README in my Github.

File: repository/p.json [hunk 1]
```diff
@@ -177,6 +177,16 @@
...
 		{
+			"name": "PandocsMarkdown",
+			"details": "https://github.com/geniusupgrader/pandocsmarkdown",
+			"releases": [
+				{
+					"sublime_text": "*",
+					"tags": true
+				}
+			]
+		}
```
Comment (FichteFoll) [inline repository/p.json]: There is a comma missing here.

**Comment by FichteFoll** [comment]
Your latest tag `3.0.0` won't be picked up because semantic versions consist of exactly 3 numeric parts.

You also have `import Pandown.minify_json as minify_json` but don't state "Pandown" as being a requirement to run this package.

"PandocsMarkdown" defines a `.sublime-syntax` file, which are only available from build 3092 onwards. You should restrict installation to these builds.

**Comment by geniusupgrader** [comment]
My package "PandocsMarkdown" is obsolete now. I will contribute to Markdown Extended.
But for "Spandoc" I will try to eliminate the failures and the requirements.
Maybe this should be closed? I will try again...
Thanks

**Comment by FichteFoll** [comment]
No need to close it. Just remove the redundant package in a commit.

**Comment by geniusupgrader** [comment]
I think, I've remedied all your requested changes. Please merge it, if there are no more failures.

**Comment by geniusupgrader** [comment]
I have deleted the messages folder

**Comment by geniusupgrader** [comment]
Now, everything is clear, I guess...

**Pull request merged**